pkgs/servers/misc/navidrome/update.nix

```diff
@@ -22,7 +22,7 @@ writeScript "update-navidrome" ''
   tempDir=$(mktemp -d)
   cp $src/ui/package.json $src/ui/package-lock.json $tempDir
   cd $tempDir
-  node2nix -l package-lock.json -c node-composition.nix
+  node2nix -d -l package-lock.json -c node-composition.nix
```

Comment:
Why is `-d` needed?

Reply:
The "react-scripts" dependency is needed to build. Upstream moved this dependency to the "devDependencies" section in package.json. But "devDependencies" only get installed with the "-d" option in node2nix. So that had to be added.

https://github.com/navidrome/navidrome/commit/b1b081e3d80

Reply:
Thanks for satisfying my curiosity :)